estate/models/estate_property_type.py

```diff
@@ -0,0 +1,8 @@
+from odoo import fields, models
+
+
+class EstatePropertyType(models.Model):
+    _name = "estate.property.type"
+    _description = "The model estate.property.type has no access rules, consider adding one."
```

Comment:
This is the model name but user-friendly

```suggestion
    _description = "Estate Property Type"
```

Please fix this in all the same occurrences.